pkg/entrypoint/entrypoint.go

```diff
@@ -90,7 +89,7 @@ func startAllServicesWithParameters(opts startupOptions) *ServiceContext {
 	// start services
 	log.Log(log.Entrypoint).Info("ServiceContext start scheduling services")
 	sched.StartService(eventHandler, opts.manualScheduleFlag)
-	proxy.StartService(eventHandler)
+	proxy.StartService()
```

Comment:
I'm thinking about the necessity of having handler.EventHandlers struct.

Not sure why we need it...  To me, I think it would also works if we remove  handler.EventHandlers and change the startService() function to:
```
	sched.StartService(proxy, opts.manualScheduleFlag)
	proxy.StartService()
```

I suppose the reason for having have handler.EventHandlers is to group all EventHandler objects.  However, if we implement this change... the handler.EventHandlers will be reduntant since it's only existing for RMProxyEventHandler .

Reply:
Yeah that's correct. I think we can fix this in a follow-up JIRA.